### PR TITLE
[15.0][MIG] edi_voxel_secondary_unit > edi_voxel_secondary_unit_oca: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36458

ping @carlosdauden @CarlosRoca13 

Depends on:
- [x] https://github.com/OCA/edi/pull/676

### CODE REVIEW

File: edi_voxel_secondary_unit_oca/views/product_secondary_unit_views.xml [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
 <odoo>
-    <record id="product_secondary_unit_view_tree" model="ir.ui.view">
-        <field name="name">product.secondary.unit.view.tree.inherit</field>
-        <field name="model">product.secondary.unit</field>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
         <field
             name="inherit_id"
-            ref="product_secondary_unit.product_secondary_unit_view_tree"
+            ref="product_secondary_unit.product_template_form_view"
```
Comment (carlosdauden) [inline edi_voxel_secondary_unit_oca/views/product_secondary_unit_views.xml:9]: I assume that some changes have been made in the base module that cause the change from one view to another

**Review by carlosdauden — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Rebase done!!

**Comment by carlosdauden** [comment]
Tests fails because:
edi_voxel_secondary_unit_oca (Production/Stable) depends on edi_voxel_oca (Beta)

**Comment by pedrobaeza** [comment]
The problem is:

`edi_voxel_secondary_unit_oca (Production/Stable) depends on edi_voxel_oca (Beta)`

**Pull request merged**